yunwoo/baekjoon/자료구조/1920-수-찾기.js

```diff
@@ -0,0 +1,41 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const integersN = new Set(input[1].split(" "));
+  const integersM = input[3].split(" ");
+  let result = "";
+
+  integersM.forEach((integer) => {
+    result += integersN.has(integer) ? "1\n" : "0\n";
+  });
+
+  console.log(result);
```

Comment:
```suggestion
  const integersN = new Set(input[1].split(" "));
  const integersM = input[3].split(" ");
  let result = "";

  integersM.forEach((integer) => {
    result += integersN.has(integer) ? "1\n" : "0\n";
  });

  console.log(result);
```
- 풀이잘봤습니다 🙇‍♂️
- integersN과 intergersM이 서로 자료구조가 Set, Array라는 점에서 차이가 있는데 네이밍이 동일해서 아쉽네요
- 차라리 integersN은 Set자료구조를 가졌기 때문에 intergerSet, 요소가 포함되어있는지 비교해야되는 integersM을 compareList 네이밍은 어떠실까요 
- 지속적으로 말씀드리지만 변수, 함수 네이밍에대해서 좀 더 고민해보시면 좋을 것 같습니다 !

Reply:
- 이름에 대해 계속 고민은 해보고있는데.. 떠오르질 않네요🥲 코멘트 감사합니다 🙇‍♂️